### PR TITLE
the exit code of the command should be returned.

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

File: shakedown/dcos/command.py [hunk 1]
```diff
@@ -51,8 +51,8 @@ def run_command(
 
         try_close(channel)
         try_close(transport)
-
-        return True
+
```
Comment (sschneid) [inline shakedown/dcos/command.py]: Looks like some extra spacing was accidentally added here.

**Comment by sschneid** [comment]
+1 aside from spacing comment above.  :)

**Comment by kensipe** [comment]
thanks!  fixed the spacing... merging on test completion.

File: shakedown/dcos/command.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def run_command(
         try_close(channel)
         try_close(transport)
 
-        return True
+        return exit_code == 0
```
Comment (colin-msphere) [inline shakedown/dcos/command.py]: Now that we have an exit code, should we just pass it back to the caller?

/cc @sschneid

Comment (sschneid) [inline shakedown/dcos/command.py]: > Now that we have an exit code, should we just pass it back to the caller?

I thought of this too, just returning `exit_code` -- I'll defer to @kensipe.

**Pull request merged**